test/index.js

```diff
@@ -100,7 +100,7 @@ describe('Drip', function(){
       test
         .set({ account: 1, token: 'x' })
         .identify(helpers.identify())
-        .error('Drip: bad request status=401 msg=Authentication failed, check your credentials', done);
```

Comment:
i remember i had this issue with klaviyo. i ran `make clean` and just removed it. nathan said it was fine since the "Drip:" is really not what we are testing against, just the response. this passes locally and in circleci
@f2prateek